lib/ClimaCoreTempestRemap/test/runtests.jl

```diff
@@ -0,0 +1,53 @@
+import ClimaCore
+using ClimaCore: Geometry, Meshes, Domains, Topologies, Spaces
+using NCDatasets
+using TempestRemap_jll
+using Test
+using ClimaCoreTempestRemap
+
+OUTPUT_DIR = mkpath(get(ENV, "CI_OUTPUT_DIR", tempname()))
+
+@testset "mesh coordinates ClimaCore > TempestRemap" begin
+    # generate CC mesh
+    ne = 9
+    R = 1
+    domain = Domains.SphereDomain(R)
+    mesh = Meshes.EquiangularCubedSphere(domain, ne)
+    topology = Topologies.Topology2D(mesh)
+    meshfile_cc = joinpath(OUTPUT_DIR, "test_cc.g")
+    write_exodus(meshfile_cc, topology)
+
+    # generate TR mesh
+    meshfile_tr = joinpath(OUTPUT_DIR, "test_tr.g")
+    run(
+        `$(TempestRemap_jll.GenerateCSMesh_exe()) --res $ne --alt --file $meshfile_tr`,
+    )
+    # tempest remap-generated mesh should have the same number of elements
+    @test Meshes.nelements(mesh) ==
+          NCDataset(nc -> nc.dim["num_elem"], meshfile_tr)
+
+    # compute overlap mesh
+    meshfile_overlap = joinpath(OUTPUT_DIR, "test_overlap.g")
+    run(
+        `$(TempestRemap_jll.GenerateOverlapMesh_exe()) --a $meshfile_cc --b $meshfile_tr --out $meshfile_overlap`,
+    )
+    # the overlap mesh should have the same number of elements as the input meshes
+    @test Meshes.nelements(mesh) ==
+          NCDataset(nc -> nc.dim["num_elem"], meshfile_overlap)
+
+    # compute weight file
+    weightfile = joinpath(OUTPUT_DIR, "test_weight.nc")
+    run(
+        `$(TempestRemap_jll.GenerateOfflineMap_exe()) --in_mesh $meshfile_cc --out_mesh $meshfile_tr --ov_mesh $meshfile_overlap --in_np 1 --out_map $weightfile`,
+    )
+    # fractions should be close to 1
+    frac_a = NCDataset(nc -> Array(nc["frac_a"]), weightfile)
+    @test all(x -> x ≈ 1, frac_a)
+
+    @test_broken begin
+        run(
+            `$(TempestRemap_jll.GenerateOfflineMap_exe()) --in_mesh $meshfile_cc --out_mesh $meshfile_tr --ov_mesh $meshfile_overlap --in_np 2 --out_map $weightfile`,
+        )
+        true
+    end
```

Comment:
@jakebolewski when we run this one, it terminates with an error return code. However when I built it locally it seemed to run fine. Any suggestions on how we can debug this?

Reply:
Don’t we still need to use LOAD_PATH for thing in runtests.jl?

Reply:
How are you running locally?

Reply:
> Don’t we still need to use LOAD_PATH for thing in runtests.jl?

No, we set up an environment for this.

> How are you running locally?

if you `]dev ../..` you can load the package locally.

Reply:
hmm if it works locally but not on CI it might be an undefined behavior bug in tempest that differs between the gcc / clang toolchains.  I can build it locally and see

Reply:
maybe it isn't clear but the process from the above comment but the process returns a 11 exit code which means it is segfaulting, I can reproduce this locally on linux with the latest JLL release
.

Reply:
Yes, that's what I meant: the JLL segfaults on both Linux and OS X, but the non-JLL binaries work fine.

Reply:
by non-jll you mean the version of Clang on your laptop?

Reply:
I'm wondering if the bug is with libblastrampoline.  I had to switch to using MKL in the otther build because of bugs encountered.

Reply:
Thanks, that would be my first guess as well.

Reply:
https://github.com/JuliaPackaging/Yggdrasil/pull/4315 for the workaround (tested locally).

Reply:
the rebuilt package is now in the registry so the min-bounds should be set to `v2.1.2+1`

Reply:
Unfortunately it doesn't look like you can use builds numbers in bounds patches.